integrations/tensorflow/iree_tf_compiler/TFL/test/flex_ops.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: iree-opt-tflite -split-input-file -iree-tflite-import-pipeline %s | IreeFileCheck %s
+
+// CHECK-LABEL: @test_flex_ops
+func @test_flex_ops(%arg0: tensor<?x2x64xf32>, %arg1: tensor<1x1x64xf32>) -> tensor<*xf32> {
+  // CHECK: %[[ADD:.+]] = "tosa.add"(%arg0, %arg1) : (tensor<?x2x64xf32>, tensor<1x1x64xf32>)
+  // CHECK: %[[CAST:.+]] = tensor.cast %[[ADD]]
+  // CHECK: return %[[CAST]]
+  %0 = "tfl.custom"(%arg0, %arg1) {custom_code = "FlexAddV2", custom_option = opaque<"tfl", "0x054164645632002E120541646456321A001A002A070A01541202300132180A16726573696475616C5F626C6F636B5F2E5F302F616464000237311414042801"> : tensor<63xi8>} : (tensor<?x2x64xf32>, tensor<1x1x64xf32>) -> tensor<*xf32>
```

Comment:
Ewwwww. Is this the only way we have to encode this? These sorts of magic strings are really hard to understand or maintain (I complained about this in TF as well). Can we at least have a comment for how someone might be able to regenerate this? Might also be nice to add some line breaks here

Reply:
I updated and added a comment about generating this. The only mechanism I would recommend is either compiling a flex op model or finding an existing one. The string is a serialized TensorFlow Node proto so it is not reasonable to be handwritten. I wish there was an alternate way but we are a bit stuck with TF's quirks.

Reply:
Yeah :-/ Thanks for adding a comment. Any chance it could point to a specific model file or something?

Reply:
Added a link to the tfhub page.